modules/manage/partials/mountable-topics.adoc

```diff
@@ -7,7 +7,7 @@ For topics with Tiered Storage enabled, you can unmount a topic to safely detach
 
 == Unmount a topic from a cluster to object storage
 
-When you unmount a topic, all incoming writes to the topic are blocked as Redpanda unmounts the topic from the cluster to object storage. Producers and consumers of the topic receive an error message indicating that the topic is no longer available. 
+When you unmount a topic, all incoming writes to the topic are blocked as Redpanda unmounts the topic from the cluster to object storage. Producers and consumers of the topic receive a warning `Failed to download manifest for topic` indicating that the topic is no longer available. 
```

Comment:
Could you change to lower case `f` in `failed` please? I would imagine them using `grep`, which is case-sensitive by default.

Reply:
And change "indicating that the topic is no longer available" to "indicating that either the topic is unavailable or there are multiple topics under the specified name" or something like this. It will be clear from the rest of the message which one is the case.

Reply:
@bashtanov yes, thank you!

Reply:
@bashtanov actually, would "multiple topics under the specified name" ever apply in the case of unmount?

Reply:
I messed up everything. This error -- `failed to download manifest for topic` -- is for mounting when the topic is not available or cannot be unambiguously defined. It will be in logs.

As for producing into a topic that is about to be unmounted, it is `invalid_topic_exception` or `resource_is_being_migrated` they will be getting. When fetching from a not-yet-ready topic it'll be `invalid_topic_exception` as well. These will be in the protocol replies.